README.md

```diff
@@ -1,6 +1,6 @@
 # IBM Spectrum Scale container native
 
-IBM Spectrum Scale in containers allows the deployment of the cluster file system in a Red Hat® OpenShift® cluster. Using a remote mount attached file system, the IBM Spectrum Scale solution provides a persistent data store to be accessed by the applications via the [IBM Spectrum Scale Container Storage Interface (CSI) driver](https://www.ibm.com/docs/en/spectrum-scale-csi?topic=spectrum-scale-container-storage-interface-driver-231) using Persistent Volumes (PVs).
+IBM Spectrum Scale in containers allows the deployment of the cluster file system in a Red Hat® OpenShift® cluster. Using a remote mount attached file system, the IBM Spectrum Scale solution provides a persistent data store to be accessed by the applications via the [IBM Spectrum Scale Container Storage Interface (CSI) driver](https://www.ibm.com/support/knowledgecenter/STXKQY_CSI_SHR/com.ibm.spectrum.scale.csi.v2r10.doc/bl1csi_kc_landing.html) using Persistent Volumes (PVs).
```

Comment:
https://www.ibm.com/docs/en/spectrum-scale-csi?topic=spectrum-scale-container-storage-interface-driver-231

This needs to be updated at the very least to the current release of CSI that is in tandem with CNSA, for the case of 5.1.2.1 it should be 2.4.0. 
However I wonder if we should link to the top level CSI so it's not so needy from release to release.

Reply:
I pushed a commit to land on the top level here: https://www.ibm.com/docs/en/spectrum-scale-csi